src/main/java/monero/wallet/MoneroWalletFull.java

```diff
@@ -20,1870 +20,1872 @@
-  
-  private static List<MoneroTransfer> deserializeTransfers(MoneroTransferQuery query, String blocksJson) {
-    
-    // deserialize blocks
-    DeserializedBlocksContainer deserializedBlocks = deserializeBlocks(blocksJson);
-    List<MoneroBlock> blocks = deserializedBlocks.blocks;
-    
-    // collect transfers
-    List<MoneroTransfer> transfers = new ArrayList<MoneroTransfer>();
-    for (MoneroBlock block : blocks) {
-      sanitizeBlock(block);
-      for (MoneroTx tx : block.getTxs()) {
-        if (block.getHeight() == null) tx.setBlock(null); // dereference placeholder block for unconfirmed txs
-        MoneroTxWallet txWallet = (MoneroTxWallet) tx;
-        if (txWallet.getOutgoingTransfer() != null) transfers.add(txWallet.getOutgoingTransfer());
-        if (txWallet.getIncomingTransfers() != null) {
-          for (MoneroIncomingTransfer transfer : txWallet.getIncomingTransfers()) transfers.add(transfer);
-        }
-      }
-    }
-    return transfers;
-  }
-  
-  private static List<MoneroOutputWallet> deserializeOutputs(MoneroOutputQuery query, String blocksJson) {
-    
-    // deserialize blocks
-    DeserializedBlocksContainer deserializedBlocks = deserializeBlocks(blocksJson);
-    List<MoneroBlock> blocks = deserializedBlocks.blocks;
-    
-    // collect outputs
-    List<MoneroOutputWallet> outputs = new ArrayList<MoneroOutputWallet>();
-    for (MoneroBlock block : blocks) {
-      sanitizeBlock(block);
-      for (MoneroTx tx : block.getTxs()) {
-        outputs.addAll(((MoneroTxWallet) tx).getOutputsWallet());
-      }
-    }
-    return outputs;
-  }
-  
-  private static class AddressBookEntriesContainer {
-    public List<MoneroAddressBookEntry> entries;
-  }
-  
-  // ---------------------------- PRIVATE HELPERS -----------------------------
-  
-  /**
-   * Enables or disables listening in the c++ wallet.
-   */
-  private void refreshListening() {
-    boolean isEnabled = listeners.size() > 0;
-    if (jniListenerHandle == 0 && !isEnabled || jniListenerHandle > 0 && isEnabled) return; // no difference
-    jniListenerHandle = setListenerJni(isEnabled ? jniListener : null);
-  }
-  
-  private void assertNotClosed() {
-    if (isClosed) throw new MoneroError("Wallet is closed");
-  }
-  
-  private static MoneroAccount sanitizeAccount(MoneroAccount account) {
-    if (account.getSubaddresses() != null) {
-      for (MoneroSubaddress subaddress : account.getSubaddresses()) sanitizeSubaddress(subaddress);
-    }
-    return account;
-  }
-  
-  private static MoneroSubaddress sanitizeSubaddress(MoneroSubaddress subaddress) {
-    if ("".equals(subaddress.getLabel())) subaddress.setLabel(null);
-    return subaddress;
-  }
-  
-  private static MoneroBlock sanitizeBlock(MoneroBlock block) {
-    for (MoneroTx tx : block.getTxs()) sanitizeTxWallet((MoneroTxWallet) tx);
-    return block;
-  }
-  
-  private static MoneroTxWallet sanitizeTxWallet(MoneroTxWallet tx) {
-    return tx;
-  }
-}
+ package monero.wallet;
```

Comment:
The indentation of this entire class needs reverted so diffs are not appearing for every line.